src/Compatibility/Microsoft.DotNet.Compatibility/ValidationLogger.cs

```diff
@@ -4,20 +4,21 @@
 using System.IO;
 using Microsoft.Build.Framework;
 using Microsoft.DotNet.Compatibility.ErrorSuppression;
+using Microsoft.DotNet.PackageValidation;
 using Microsoft.NET.Build.Tasks;
 
-namespace Microsoft.DotNet.PackageValidation
+namespace Microsoft.DotNet.Compatibility
 {
-    internal class PackageValidationLogger : IPackageLogger
+    internal class ValidationLogger : ICompatibilityLogger
```

Comment:
NIT: Validation seems like an overloaded term to me. Perhaps CompatibilityLogger would be more accurate.